plugins/registry.json

```diff
@@ -255,5 +255,11 @@
         "title": "Scam Link Detector",
         "icon_url": "https://cdn.discordapp.com/attachments/576521645540245505/895661244743299102/antifish.png",
         "thumbnail_url": "https://cdn.discordapp.com/attachments/576521645540245505/895661244743299102/antifish.png"
+    },
+    "raw-content": {
+        "repository": "Nzii3/modmail-plugins",
+        "branch": "main",
+        "description": "Have the ability to get raw embed description (useful for iOS mobile users).",
+        "title": "Raw Content",
```

Comment:
As mentioned by Stephen, it's preferred to include an icon and thumbnail (given by URL). Also, I don't believe you're allowed to end JSON objects with a trailing comma, that would probably trigger a parsing error.

Reply:
I've fixed the comma issue, but would still prefer to have the two images.